### PR TITLE
Add fiat on ramps

### LINKED ISSUE #901: Integrate Kado
May be as simple as opening an iframe with:
```
https://app.kado.money/?onPayCurrency=USD&onPayAmount=200&onRevCurrency=USDC&offPayCurrency=USDC&offRevCurrency=USD&network=OSMOSIS&onToAddress=X&offFromAddress=X&cryptoList=USDC&networkList=OSMOSIS&apiKey=67a2aaad-a2a5-4412-be40-e3a70aa7b53d&product=BUY
```

### LINKED ISSUE #902: Integrate Transak

### PR DESCRIPTION
**Pull request by jonator**
* Closes #901 
* Closes #902

### CODE REVIEW

File: packages/web/integrations/kado/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+> = ({ assetKey }) => {
+  const { chainStore, accountStore } = useStore();
+
+  const account = accountStore.getAccount(chainStore.osmosis.chainId);
+
+  if (!(account.walletStatus === WalletStatus.Loaded)) return null;
+
+  return (
+    <iframe
+      src={`https://app.kado.money/?onPayCurrency=USD&onPayAmount=200&onRevCurrency=${assetKey}&offPayCurrency=${assetKey}&offRevCurrency=USD&network=OSMOSIS&onToAddress=${account.bech32Address}&offFromAddress=X&cryptoList=${assetKey}&networkList=OSMOSIS&apiKey=67a2aaad-a2a5-4412-be40-e3a70aa7b53d&product=BUY`}
```
Comment (delivan) [inline packages/web/integrations/kado/index.tsx:18]: 1. Is it okay to show api key publicly?
2. I've heard that there are some iframe security concerns. how's your thinking about using iframe?

**Comment by delivan** [comment]
@jonator Can you add Transak API key as environment variable in Vercel to test on preview?

File: packages/web/integrations/kado/index.tsx (hunk 1 above)
Comment (jonator) [reply packages/web/integrations/kado/index.tsx:18]: 1. I've confirmed with Kado team that it is just a public identifier
2. Any specifics about the concern? This is the recommended method of integration by Kado

**Comment by jonator** [comment]
@delivan Done, though I may remove it if Transak responds that it doesn't have to stay secret. To keep deployments simple

File: packages/web/integrations/kado/index.tsx (hunk 1 above)
Comment (delivan) [reply packages/web/integrations/kado/index.tsx:18]: 2. I saw [this article](https://blogs.halodoc.io/iframe-security-threats-and-the-prevention/) but I think Kado is a trusted third party. Ignore it.

**Comment by delivan** [comment]
1. The close button doesn't look like a close button
2. Modal isn't closed when clicking "Exit Now" button
스크린샷 2022-10-26 오후 11 52 54
스크린샷 2022-10-26 오후 11 52 59

**Comment by jonator** [comment]
Carlo from Transak told me it's fine to not keep prod API key secret

**Comment by jonator** [comment]
Hmm, modal closes for me. I told Transak team about close button issue as well as a few other issues.

**Comment by jonator** [comment]
Currently blocked on Kelsey setting up an auto forwarding rule for the dashboard email alias for an HR email so I can get verification codes

**Pull request merged**